packages/astro/src/core/config.ts

```diff
@@ -34,6 +34,7 @@ const ASTRO_CONFIG_DEFAULTS: AstroUserConfig & any = {
 	server: {
 		host: false,
 		port: 3000,
+		streaming: true,
```

Comment:
Yeah I wasn't quite sure about this one either, it just felt like a bit better DX to disable streaming on the server with `config.server.streaming` 🤷 

Happy to move it to build if that makes the most sense

Reply:
I agree `server.streaming` sounds a lot nicer, but it's just conceptually not what this config object is designed for right now. If it helps, you could think of it more as `config.devServerOptions` and `config.buildOutputOptions`.

Renaming `server` to `devServer` or something more clear could make sense as a seperate convo.

Reply:
Oh! I just realized that right now the dev server is accepting this option. I'd say that `dev` and `preview` should always stream, since we know that our dev server and localhost support it. 

If a user reports a bug with this then I'm happy to revisit, but I feel pretty strongly we should start from a place of "default streaming" with response buffering as a deploy-specific fallback.

Reply:
I actually did that on purpose, I'm always a little hesitant to have dev and production builds act differently.  Agreed it makes sense to only use the flag in production if it's moved to `build` config though, I'll get that updated :+1: